packages/api/src/server.ts

```diff
@@ -130,9 +131,13 @@ export class WepublishServer {
       cors: corsOptions
     })
 
-    app.use((err: any, req: Express.Request, res: Express.Response, next: NextFunction) => {
+    app.use((err: any, req: Request, res: Response, next: NextFunction) => {
       logger('server').error(err)
-      return next(err)
+
+      if (err.status === 413) {
+        res.status(413).send({error: `Payload cannot exceed ${MAX_PAYLOAD_SIZE}`})
+        return next(err)
```

Comment:
It logs to the console. The error in the editor is "Response not successful: Received status code 413" instead of the previous error about '<'.

Reply:
@tomaszdurka done and looking forward to feedback. I left out the 413 custom message because the generic message in console contains more information. Editor error message is the same either way.